examples/python/arkit_scenes/main.py

```diff
@@ -402,7 +402,7 @@ def log_arkit(recording_path: Path, include_highres: bool) -> None:
                 colors_list,
             )
 
-            rr.log(f"{lowres_posed_entity_id}/rgb", rr.Image(rr.TensorData(array=rgb, jpeg_quality=95)))
+            rr.log(f"{lowres_posed_entity_id}/rgb", rr.Image(rgb).compress())
```

Comment:
I think we should be explicit in our own code at least, even if we let users omit the the `jpeg_quality` argument

```suggestion
            rr.log(f"{lowres_posed_entity_id}/rgb", rr.Image(rgb).compress(jpeg_quality=95))
```